KomaMRIBase/test/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
```

Comment:
:eyes:

Reply:
Addresed in https://github.com/cncastillo/KomaMRI.jl/commit/e37397a7c4d95cc942a71195de6c52d1aeacfe01